api/openapi.yaml

```diff
@@ -130,7 +131,6 @@ paths:
                 $ref: "./transactions.yaml#/components/schemas/Transaction"
         400:
           description: "trying to create a transaction with an already existing ID"
-
```

Comment:
Removed some blank lines our OpenAPI to be easy to collapse it and have a better view of the endpoints, like this: 
<img width="899" alt="image" src="https://user-images.githubusercontent.com/12397879/223792184-94ee22fe-ca39-4f1b-919d-a1d282a28074.png">